spec/support/shared_contexts/api/filterable_endpoint.rb

```diff
@@ -46,27 +46,97 @@
 
   it "returns only resource that have been updated since the provided date" do
     updated_since = 2.months.ago.utc.iso8601
-    authenticated_api_get(path, params: { filter: { updated_since: } })
+    params = { filter: { updated_since: } }
+    params.deep_merge!(mandatory_params) if defined?(mandatory_params)
+    authenticated_api_get(path, params:)
 
     expect(response).to have_http_status(:ok)
     expect(response.content_type).to eql("application/json; charset=utf-8")
-    expect(response.body).to eq(serializer.render(apply_expected_order([resource_updated_one_week_ago, resource_updated_one_month_ago]), root: "data"))
```

Comment:
sorry Leandro I misunderstood we were using serializer here, can we add it back please as makes sense instead of the id, I think the only thing we don't need was the query, the serializer we can keep, sorry!